File: FCUBEAPI/Consignment/Models/DoTempGcListModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Consignment.Models
{
    public class DoTempGcListModel
    {
        public List<DoTempGcModel> DoTempGcList { get; set; }
        public PaginationMetaData PageMetaData { get; set; }
    }

}

File: FCUBEAPP/src/app/models/dotempgclistmodel.ts
import { Paginationmodel } from "./paginationmodel";
import { Dotempgcmodel } from "./dotempgcmodel";

export class Dotempgclistmodel {
    doTempGcList: Dotempgcmodel[] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}
